recipe/meta.yaml

```
@@ -30,14 +30,13 @@ requirements:
     - m2-automake{{ am_version }}  # [win]
     - m2-libtool  # [win]
     - m2w64-pkg-config  # [win]
-    - make
     - pkg-config  # [not win]
     - posix  # [win]
     - gnuconfig  # [unix]
-    - {{ stdlib("c") }}
-    - {{ compiler('c') }}        # [unix]
-    - {{ compiler('m2w64_c') }}  # [win]
-    - make                          # [unix]
+    - make
+    - {{ stdlib('c') }}             # [unix]
+    - {{ compiler('c') }}           # [unix]
+    - {{ compiler('m2w64_c') }}     # [win]
```

```suggestion
    - {{ compiler('m2w64_c') }}     # [win]
    - {{ stdlib('m2w64_c') }}       # [win]
```

it seems tha tthis is needed.

---

I'm just leaving this here since i know you made many similar PRs so you will have to fix them there too. Otherwise I would push and merge.